packages/editor-ui/src/router.ts

```diff
@@ -293,6 +294,28 @@ export const routes = [
 			},
 		],
 	},
+	{
+		path: '/workflow/:name/history/:id?',
```

Comment:
Why `:name`? Won't this be the ID?

Reply:
Just copying previous routes, `NodeView.vue` uses it like this but since this component will not use I can probably rename